### DISCUSSION TITLE
need help

### DISCUSSION THREAD

**Discussion by AgostinaSarasua** [community/community#131445] [GitHub Education]
### Hi everyone,

Hello how are you? I created an account in GutHub Developer Pack, 2 years ago and today I need to log in again to be able to download a program. When confirming my data, it asks me to take a photo of a study certificate that proves that I am studying at an institution, but the photo is blurry and I would like to send GitHub a proof in PDF format but it does not give me that option, Does anyone know how I can upload a file in PDF format that is my study certificate?

**Comment by davevad93** [comment]
Hi @AgostinaSarasua ,

Unfortunately, it's not possible to upload a PDF file as proof to get the Student Developer Pack, only pictures are allowed. When I applied last year, there was an option to upload a picture instead of taking a webcam photo, but I've seen many users say that right now the only available option is taking a photo.

**Reply by AgostinaSarasua** [reply]
Thank you!

**Comment by trikrapka** [comment]
Unfortunately it is not possible to upload a picture. I mean you can upload it as png/jpg, at least it was possible a year ago or so, but they will say something like "you should take a picture using your device camera". 
So i'd recommend you to try to take the picture with more light, it will reduce blurriness

**Reply by AgostinaSarasua** [reply]
Thank you!
